packages/frontend/src/pages/help-page/components/help-sidebar/help-sidebar.tsx

```diff
@@ -14,33 +14,46 @@
  * limitations under the License.
  */
 
-import { Badge, Box, BoxProps, List, ListItem, VStack } from '@chakra-ui/react';
+import {
+  Badge,
+  Box,
+  BoxProps,
+  Drawer,
+  DrawerBody,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerHeader,
+  DrawerOverlay,
+  IconButton,
+  List,
+  ListItem,
+  useColorModeValue,
+  useDisclosure,
+  VStack
+} from '@chakra-ui/react';
 
 import { SidebarHeading } from '../../../../components/sidebar-heading/sidebar-heading';
 import { SidebarStack } from '../../../../components/sidebar-stack/sidebar-stack';
+import { iconFactoryAs } from '../../../../shared/icon-factory';
 
 type HelpSidebarLink = { name: string; hash: string; children?: HelpSidebarLink[]; iex?: boolean };
 
 interface Props {
   links: HelpSidebarLink[];
+  headerTitle?: string;
 }
 
 const IexBadge = () => <Badge variant="frost">IEX</Badge>;
 
-export const HelpSidebar = ({ links, ...props }: Props & BoxProps) => {
+const SidebarContent = ({ links }) => {
   return (
-    <VStack
-      spacing="1rem"
-      align="stretch"
-      flexBasis={{ base: '16rem', md: '20rem', xl: '22rem' }}
-      flexShrink={0}
-      maxWidth={{ base: '16rem', md: '20rem', xl: '26rem' }}
-      ml="1rem"
-      mt="1rem"
-      pt="1rem"
-      {...props}
-    >
-      <Box position="sticky" top="1rem" overflowY="auto" maxHeight="100vh">
+    <>
```

Comment:
You just have one component here (the `<Box>`), so you don't need to wrap it in `<></>`.